packages/dso-toolkit/src/components/plekinfo-card/plekinfo-card.args.ts

```diff
@@ -0,0 +1,63 @@
+import { HandlerFunction } from "@storybook/addon-actions";
+import { ArgTypes } from "@storybook/types";
+
+import { noControl } from "../../storybook";
+import { PlekinfoCard, PlekinfoWijzigactie } from "./plekinfo-card.models.js";
+import { Renvooi } from "../renvooi/renvooi.models.js";
+import { Label } from "../label/label.models.js";
+
+export interface PlekinfoCardArgs {
+  label: Renvooi | string;
+  href: string;
+  active: boolean;
+  meta?: Label;
+  wijzigactie?: PlekinfoWijzigactie;
+  dsoPlekinfoCardClick?: HandlerFunction;
+}
+
+export const plekinfoCardArgs: PlekinfoCardArgs = { href: "#", label: "Radarverstorende bouwwerken", active: false };
+
+export const plekinfoCardArgTypes: ArgTypes<PlekinfoCardArgs> = {
+  label: {
+    control: {
+      type: "text",
```

Comment:
@tfrijsewijk

De typering van deze control gaat in de nog niet goed. type: "text" werkt natuurlijk goed samen met type string van `label` in `PlekinfoCardArgs`, echter hoe deal ik met type `Renvooi`? 

![Screenshot 2024-10-07 at 08 26 41](https://github.com/user-attachments/assets/8c27d643-5be4-409c-b9eb-0ae5d0439aef)

Reply:
Tsja, dat is inderdaad lastig. Ik denk dat je `type: "object"` nodig hebt: https://storybook.js.org/docs/essentials/controls#annotation

Reply:
Gedaan